enterprise/backend/test/metabase_enterprise/sso/integrations/jwt_test.clj

```diff
@@ -334,3 +335,25 @@
           clojure.lang.ExceptionInfo
           #"Sorry, but you'll need a test account to view this page. Please contact your administrator."
           (#'mt.jwt/fetch-or-create-user! "Test" "User" "test1234@metabase.com" nil)))))))
+
+(deftest get-jwt-token-test
+  (testing "should return a session token when token=true"
```

Comment:
worth testing the `token=false` case just in case?

Reply:
that's a good idea too